src/components/delete-account/delete-account-controller.ts

```diff
@@ -56,7 +56,7 @@ export function deleteAccountPost(
 
     const DeleteTopicARN = getSNSDeleteTopic();
     try {
-      await service.deleteServiceData(subjectId, DeleteTopicARN);
+      await service.deleteServiceData(publicSubjectId, DeleteTopicARN);
```

Comment:
this is for deleting service Data which uses the subject_id . I think the issue was just the logging 
https://github.com/alphagov/di-account-management-frontend/blob/main/src/components/delete-account/delete-account-controller.ts#L80 (which refers to the subject_id instead of public subject_id)

Reply:
Ah. So my testing did work but i wasn't testing the right thing. 
Glad you understand it.  thank you.   I've updated the PR now

Reply:
Thank you :) Should we also do it here 
https://github.com/alphagov/di-account-management-frontend/blob/f8638914456e6252ff50da3dc07c6ecaf9eb6ba8/src/components/check-your-email/check-your-email-controller.ts#L64